alma-python3.9/Dockerfile

```diff
@@ -0,0 +1,92 @@
+#
+# Copyright (C) 2018-2021 CERN.
+# Copyright (C) 2022 Graz University of Technology.
+# Copyright (C) 2022 University of Münster.
+#
+# Invenio is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+#
+ARG LINUX_VERSION=9.0
+FROM almalinux:${LINUX_VERSION}
+
+RUN dnf upgrade --refresh -y && \
+    dnf install -y \
+        dnf-plugins-core \
+        git \
+        tmux \
+        vim \
+        wget \
+        emacs-nox  \
+        glibc-common \
+        glibc-locale-source \
+        glibc-langpack-en
+
+RUN localedef -i en_US -c -f UTF-8 -A /usr/share/locale/locale.alias en_US.UTF-8 && \
+    ln -s /usr/bin/python3 /usr/bin/python
+
+ENV LANG en_US.UTF-8
+ENV LANGUAGE en_US:en
+ENV LC_ALL en_US.UTF-8
+
+# crb is needed to install epel, epel contains ImageMagick
+RUN dnf config-manager --set-enabled crb && \
+    dnf install -y \
+        https://dl.fedoraproject.org/pub/epel/epel-release-latest-9.noarch.rpm
+
+RUN dnf groupinstall -y "Development Tools" && \
+    dnf install -y \
+        pip \
+        python3-devel \
+        cairo-devel \
+        dejavu-sans-fonts \
+        libffi-devel \
+        libxml2-devel \
+        libxslt-devel \
+        ImageMagick \
+        openssl-devel \
+	    bzip2-devel \
+	    xz-devel \
+        sqlite-devel \
+        xmlsec1-devel
+
+# install nodejs version 14, nvm.sh adds functions to shell, bash -ic loads .profile first, so functions are available
+RUN curl https://raw.githubusercontent.com/creationix/nvm/master/install.sh | bash && \
```

Comment:
why `creationix`?
isn't it better the official `GitHub` repo? https://raw.githubusercontent.com/nvm-sh/nvm/master/install.sh
I am wondering if we should use a tag instead of latest master...

Reply:
Ooops, sorry, that was by accident.
What version of nvm do you suggest? 0.39.1 (latest) is from december 21 and works with every command needed for InvenioRDM. So I'm fine with tagging.